apps/web/vibes/soul/primitives/toaster/index.tsx

```diff
@@ -31,7 +32,7 @@ export const Toaster = ({ ...props }: ToasterProps) => {
 };
 
 export const toast = {
-  success: (message: string, options?: ToastOptions) => {
+  success: (message: ReactNode, options?: ToastOptions) => {
```

Comment:
For all these builtin toast methods, wouldn't we want to use something like

```tsx
toast.custom(<Alert type="error" message={<p>Message with <Link href="#">link</Link>.} />)
```

Reply:
This is not needed, unless I'm misunderstanding.

![Screenshot 2024-12-16 at 3 02 04 PM](https://github.com/user-attachments/assets/6e60dc40-e36e-4615-81de-a3b91824e11c)